Source/WebKitLegacy/mac/WebView/WebView.mm

```diff
@@ -2312,10 +2314,9 @@ - (void)_dispatchTileDidDraw:(CALayer*)tile
         return;
     }
 
-    ALLOW_DEPRECATED_DECLARATIONS_BEGIN
-    if (!OSAtomicCompareAndSwap32(0, 1, &_private->didDrawTiles))
+    BOOL old = NO;
+    if (!atomic_compare_exchange_strong((_Atomic BOOL*)&_private->didDrawTiles, &old, YES))
```

Comment:
Not my area of expertise, but I think we'd prefer using an atomic operation from `<wtf/Atomics.h>`. I see that this would be the first non-3rd-party use of `atomic_compare_exchange_strong` in the codebase.